src/core/issues-notes.js

```diff
@@ -119,7 +122,31 @@ function handleIssues($ins, ghIssues, conf) {
         }
       }
       $tit.find("span").text(text);
-      if (report.title) {
+      if (ghIssue && report.title && githubAPI) {
+        const labelsGroup = Array.from(ghIssue.labels)
+          .map(label => {
+            const issuesURL = new URL("issues/", conf.github + "/");
+            issuesURL.searchParams.set(
```

Comment:
Using `searchParams` will be problematic in IE11 (which still has security support), can we ignore it?

Reply:
Yeah, we don't support IE.